datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -83,7 +83,11 @@ def compile_queries(self):
     def execute(self, extra_tags=None):
         """This method is what you call every check run."""
         logger = self.check.log
-        global_tags = list(set(self.tags + (extra_tags or [])))
+        if extra_tags:
+            global_tags = list(extra_tags)
+            global_tags.extend(self.tags)
+        else:
+            global_tags = self.tags
```

Comment:
```suggestion
        global_tags = list(self.tags) + list(extra_tags or [])
```
It can still be a single liner without deduplication

Reply:
This way like the one on master makes a needless temporary list (or 2 in the case of no extra_tags). Could do:

```python
global_tags = self.tags + extra_tags if extra_tags else self.tags
```

but then `self.tags` and `extra_tags` must be a list rather than an arbitrary iterable like a tuple that we often use